src/interface/meta.thrift

```diff
@@ -1123,8 +1123,9 @@ struct VerifyClientVersionResp {
 
 
 struct VerifyClientVersionReq {
-    1: required binary version = common.version;
-    2: common.HostAddr host;
+    1: required binary thrift_version = common.version;
+    2: binary build_version;
+    3: common.HostAddr host;
```

Comment:
Don't modify the field number, you could add it at end.

Reply:
done